CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ Notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 Kubeclient release versioning follows [SemVer](https://semver.org/).
 
+#### past version 1.2.0
```

Comment:
This should go at the bottom, changelog is ordered newest-first.

Reply:
done (pasted it there as todo and then forgot :D )